app/controllers/home_controller.rb

```diff
@@ -0,0 +1,7 @@
+class HomeController < ApplicationController
+
+  def index
+    markets = Market.all
+    @markets = markets.order(:name)
```

Comment:
Since you are using the same `order` logic in both this controller and the markets controller, I think it would make more sense to create a method in the model that did this ordering logic that both controller could use. That way, if you ever wanted to change what the default order was, you'd only need to change it in one place.